android/app/src/main/java/app/organicmaps/sdk/routing/LaneInfo.java

```diff
@@ -0,0 +1,27 @@
+package app.organicmaps.sdk.routing;
+
+import androidx.annotation.NonNull;
+
+public final class LaneInfo
+{
+  public final LaneWay[] mLaneWays;
+  public final LaneWay mActiveLaneWay;
```

Comment:
Why do we still use Hungarian notation in the code? 😱 Does it mean that if I submit new PR, my code style should also adhere to Hungarian notation? Please, point to a code style README

Reply:
Please follow the existing code style. We may change it later, for now there's no need to introduce different style.

Reply:
Could you please share or point me to the document containing our style guidelines?